public/doc/openapi-3-v1.3.json

```diff
@@ -3088,14 +3088,14 @@
           "204": {
             "description": "Changed the sequence"
           },
-          "400": {
```

Comment:
With the removal of this line, do we only have defined responses for `401`, `403`, and `404`? Do we not need a defined response for `400`? What happens when the server is expected to return a 400 similar to what you mentioned in the JIRA ticket with the `{"errors":[{"status":"400","detail":"Neither increment nor placement parameter is set"}]}` response?

Reply:
good catch. updated